doc/lsst.analysis.ap/tasks/plotImageSubtractionCutoutsDRP.rst

```diff
@@ -0,0 +1,40 @@
+.. py:currentmodule:: lsst.analysis.ap
+
+#########################################
+Generating image cutouts with DRP outputs
+#########################################
+
+To generate image cutouts using the ``plotImageSubtractionCutouts`` task from ``analysis_ap``, use the
+code below.
+
+Note: The code differs from its general implementation when using DRP outputs since each of the data
+references need to be iterated individually. Use the ``diaSources`` from the
+``goodSeeingDiff_diaSrcTable`` which has the required dataframe format for this task.
+
+.. code-block:: python
+
+    import lsst.daf.butler as dafButler
+    from lsst.analysis.ap import plotImageSubtractionCutouts
+
+    repo = '/repo/main'
+    collections = '/path/to/your/DRP/collection'
+    butler = dafButler.Butler(repo, collections=collections)
+
+    cutoutConfigDrp = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsConfig()
+    cutoutTaskDrp = plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask(
+        config=cutoutConfigDrp, output_path='/your/output/path'
+    )
+
+    data_refs = butler.query_datasets(
+        "goodSeeingDiff_diaSrcTable", where=where, limit=limit
+    )
+
+    for ref in data_refs:
+        try:
+            dv_diaSourceTable = butler.get(ref)
+        except:
+            print(f"Could not load diaSource table for {ref.dataId}")
+            continue
```

Comment:
This `try..except` block shouldn't be necessary. `data_refs` is explicitly a list of the diaSourceTables that should exist in the Butler.